runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java

```diff
@@ -650,7 +650,7 @@ void stopPlugin(Plugin plugin, PluginListItem listItem)
 
 		if (config == null)
```

Comment:
What happens when all plugins are unpinned? I think this becomes an empty string.